plugins/scaffolder-backend-module-bitbucket/src/actions/bitbucket.examples.test.ts

```diff
@@ -268,9 +271,9 @@ describe('publish:bitbucket', () => {
     expect(initRepoAndPush).toHaveBeenCalledWith({
       dir: mockContext.workspacePath,
       remoteUrl: 'https://bitbucket.org/workspace/cloneurl',
-      auth: { username: 'x-token-auth', password: 'your-auth-token' },
+      auth: { username: 'x-token-auth', password: 'tokenlols' },
```

Comment:
I've just looked through the code and this would involve changes to tests that sit outside the scope of "examples". Ill open another PR to remove `tokenlols`

Reply:
Ok, that works for me, thanks for being open to some "extra" work 👍